qutip/solver/ode/evolver.py

```diff
@@ -0,0 +1,141 @@
+
+from ..evolver import Evolver, evolver_collection
+from .verner7efficient import vern7
+from .verner9efficient import vern9
+from .wrapper import QtOdeFuncWrapper
+import numpy as np
+from ...core import data as _data
+
+__all__ = ['EvolverVern', 'EvolverDiag']
+
+
+class EvolverVern(Evolver):
+    # -------------------------------------------------------------------------
+    # Solve an ODE for func.
+    # Calculate the required expectation values or invoke callback
+    # function at each time step.
+    #
+    # Use verner method implimented in cython
+    #
+    _error_msg = ("ODE integration error: Try to increase "
+                  "the allowed number of substeps by increasing "
+                  "the nsteps parameter in the Options class.")
+    description = "qutip implementation of verner most efficient runge-kutta"
```

Comment:
"Qutip implementation of Verner's most efficient Runge-Kutta."

Reply:
There is 2 version of Verner method for each order, 'most efficient' and 'most robust', name given by the author.
See http://people.math.sfu.ca/~jverner/.